persistence/sql/migratest/testdata/20200830154602_testdata.sql

```diff
@@ -1,5 +1,3 @@
 INSERT INTO identity_verification_tokens (id, token, used, used_at, identity_verifiable_address_id, selfservice_verification_flow_id, created_at, updated_at, expires_at, issued_at)
 VALUES ('ee56574d-2f0c-43f6-8d26-0062938ae330', '1001ba7ddd644cb68478e8947e4jfhc', false, null, '45e867e9-2745-4f16-8dd4-84334a252b61', '5385c962-0295-4575-9b1b-d7eef13c0eda', '2013-10-07 08:23:19', '2013-10-07 08:23:19', '2013-10-07 08:23:19', '2013-10-07 08:23:19');
 
-INSERT INTO identity_verification_tokens (id, token, used, used_at, identity_verifiable_address_id, selfservice_verification_flow_id, created_at, updated_at, expires_at, issued_at)
```

Comment:
Please re-add this or give a reason why it was removed

Reply:
Because the only difference between the two rows was that the `selfservice_flow_id` was null in the second, which doesn't make sense anymore, as there will always be a `selfservice_flow_id` following this change.

Reply:
Or rather, since the schema now does not allow `NULL`s in the column, this testdata statement would just fail^^

Reply:
>The last remaining piece is to add a new file to migratest/testdata to insert another row that uses your new schema :)

Also, the schema is not really "new" there is just one more not null constraint. So we already have testdata desciribing this.

Reply:
Sorry, I forgot how the testdata works. I updated the files.